wcfsetup/install/files/lib/system/html/output/HtmlOutputProcessor.class.php

```diff
@@ -90,7 +90,11 @@ public function setOutputType($outputType) {
 	 * @inheritDoc
 	 */
 	public function getHtml() {
-		return $this->getHtmlOutputNodeProcessor()->getHtml();
+		$html = $this->getHtmlOutputNodeProcessor()->getHtml();
+
+		MessageEmbeddedObjectManager::getInstance()->reset();
```

Comment:
Will this cause issues if `->getHtml()` is being called more than once for a single message?

Reply:
Good catch, this would indeed cause issues. Duh.